MAiD - Case Management App/config/project-scratch-def.json

```diff
@@ -1,7 +1,7 @@
 {
-  "orgName": "MAiD Dev",
```

Comment:
Get latest from Dev

Reply:
Scratch org def changed to be the same as dev